### PR TITLE
Dev 2018 v2

### PR DESCRIPTION
**Pull request by bokilenator**

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -36,6 +36,17 @@ def show
     @booth_chairs = @organization.booth_chairs
     @tools = Tool.checked_out_by_organization(@organization).just_tools
     @shifts = @organization.shifts
+    @user = current_user
+    @user.participant.memberships.each do |membership|
+      if membership.is_booth_chair?
+        @org_id = membership.organization_id
+      end
+    end
+    if @user.roles.map(&:name).include?("admin")
```
Comment (sclark) [inline app/controllers/organizations_controller.rb]: Let's change this to use `is_admin?` from `application_helper.rb`.

File: app/views/organization_lists/index.html.erb [hunk 2]
```diff
@@ -2,48 +2,59 @@
   <%- model_class = OrganizationList -%>
   <div class="page-header">
     <h1>
-      Add Members to Your Organization
-      <%= link_to @organization.name.possessive, @organization unless @organization.blank? %>
-      <!-- <%=t '.title', :default => model_class.model_name.human.pluralize %> -->
+      <!-- basically: if user is an admin (only admins can skip videos) -->
+      <% if can?(:skip_video, @user) %>
```
Comment (sclark) [inline app/views/organization_lists/index.html.erb]: A job for `is_admin?`

File: app/views/organization_lists/index.html.erb [hunk 3]
```diff
@@ -2,48 +2,59 @@
...
-    <tbody>
-      <% @members.each do |member| %>
-        <tr>
-          <td><%= member.organization.name %></td>
-          <td><%= member.andrew_id %></td>
-        </tr>
-      <% end %>
-    </tbody>
-  </table>
+  <% if can?(:skip_video, @user) %>
```
Comment (sclark) [inline app/views/organization_lists/index.html.erb]: Another job for `is_admin?`

File: app/views/organization_lists/index.html.erb [hunk 4]
```diff
@@ -2,48 +2,59 @@
...
+    </p>
+  <% else %>
+    <span> <strong>Your organization: </strong></span><%= @org_name %>
+  <% end %>
 
-  <p>
-    First, select your organization:
-  </p>
+  <%= form_tag add_organization_lists_path, multipart: true do%>
+    <% if can?(:skip_video, @user) %>
```
Comment (sclark) [inline app/views/organization_lists/index.html.erb]: Another job for `is_admin?`

File: app/views/organization_lists/index.html.erb [hunk 5]
```diff
@@ -2,48 +2,59 @@
...
+    </p>
+
+    <br>
+    <%= text_field_tag(:andrew_id) %>
+    <%= submit_tag "Add Member", :class => "btn btn-primary" %>
+  <% end %>
 
+  <hr>
+
+  <% if can?(:skip_video, @user) %>
```
Comment (sclark) [inline app/views/organization_lists/index.html.erb]: The final job for `is_admin?`

File: app/views/memberships/new.html.erb [hunk 6]
```diff
@@ -1,45 +1,50 @@
-<%- model_class = Membership -%>
-<div class="page-header">
-  <h1>Add Organizations for <%= @participant.name %></h1>
-</div>
+<% if can?(:skip_video, @user) %>
```
[the remark this answers is not in the document]
Comment (timparenti) [reply app/views/memberships/new.html.erb]: Per @sclark's reviews elsewhere, this line should probably just be `is_admin?`, but the lingering ability and redirect should also be addressed.

File: app/controllers/waivers_controller.rb [hunk 7]
```diff
@@ -48,9 +48,7 @@ def create
       @participant.has_signed_waiver = true
       @participant.save!
       flash[:notice] = "Thank you for completing the waiver."
-
-      @membership = Membership.new(participant: @participant)
-      render 'memberships/new'
+      redirect_to root_path
```
[the remark this answers is not in the document]
Comment (timparenti) [reply app/controllers/waivers_controller.rb]: We'd probably then want to `render 'memberships/new'` for admins, and `redirect_to root_path` with the appropriate flash notice for everyone else.

**Comment by timparenti** [comment]
I'm getting a `Table 'binder_development.organization_lists' doesn't exist` when I try to use this in development.  Was there a database migration that should have been committed?

**Comment by timparenti** [comment]
I'm getting a `Table 'binder_development.organization_lists' doesn't exist` error when I try to use this in development. Was there a database migration that should have been committed? @bokilenator

**Comment by timparenti** [comment]
I'm getting a `Table 'binder_development.organization_lists' doesn't exist` error when I try to use this in development. Was there a database migration that should have been committed? @bokilenator @sclark

**Pull request merged**